src/lib.rs

```diff
@@ -64,7 +64,7 @@ pub fn get_zones_and_stats(
             if let Some(zone) = zone::Zone::from_osm_with_geom(relation, &objects, next_index) {
                 // Ignore zone without boundary polygon for the moment
                 if zone.boundary.is_some() {
-                    stats.process(&zone);
+                    stats.init(&zone);
```

Comment:
why call it init ?

Reply:
any better name idea ? `process` seems to generic, but I agree, `init` is not really better